src/util/utility.ts

```diff
@@ -0,0 +1,4 @@
+const startValue:number=1;
+const endValue:number=90;
```

Comment:
same here see naming convention for constant value

Reply:
Resolved